10up/brightcove-video-connect
# Fix: Search box alignment

### Description of the Change
This PR fixes the search box allignment issue inside the media modal.



Closes #352 

### How to test the Change
1. Enable the Classic Editor plugin
2. Go to Post
3. Click on the "Brightcove Media" button
4. Notice that the search box is aligned properly


### Changelog Entry

> Fixed - Search box alignment in Media modal

### Credits

Props @MARQAS , @felipeelia


### Checklist:
